src/applications/disability-benefits/781/helpers.js

```diff
@@ -1,33 +1,102 @@
 import React from 'react';
 import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
 
-export const introductionText = (
+const getPtsdClassification781 = (formData, formType) => {
```

Comment:
Maybe rename this to `getPtsdClassification`, since this is for 781 and 781a.

Reply:
@imeecuison

Reply:
Changed!